tests/integration/thrift_tests.py

```diff
@@ -81,7 +83,7 @@ def on_server_span_created(self, context, server_span):
     # bind a server socket on an available port
     server_bind_endpoint = config.Endpoint("127.0.0.1:0")
     listener = make_listener(server_bind_endpoint)
-    server = make_server({"max_concurrency": "100"}, listener, processor)
+    server = make_server({"max_concurrency": "5"}, listener, processor)
```

Comment:
It's a lot of work to have to drive 100 concurrent requests, when a few will do.